collection-pipeline/src/main/java/com/iluwatar/collectionpipeline/CarFactory.java

```diff
@@ -28,19 +28,19 @@
 /**
  * A factory class to create a collection of {@link Car} instances.
  */
-public class Iterating {
-  private Iterating() {
+public class CarFactory {
+  private CarFactory() {
   }
 
   /**
    * Factory method to create a {@link List} of {@link Car} instances.
    * @return {@link List} of {@link Car}
    */
   public static List<Car> createCars() {
-    return Arrays.asList(new Car("Jeep", "Wrangler", 2011), new Car("Jeep", "Comanche", 1990),
-        new Car("Dodge", "Avenger", 2010),
-        new Car("Buick", "Cascada", 2016),
-        new Car("Ford", "Focus", 2012),
-        new Car("Chevrolet", "Geo Metro", 1992));
+    return Arrays.asList(new Car("Jeep", "Wrangler", 2011, "Jeep"), new Car("Jeep", "Comanche", 1990, "Jeep"),
+        new Car("Dodge", "Avenger", 2010, "Sedan"),
```

Comment:
@nikhilbarar The implementation seems to be going good. Just an input, we should be using Enum for car category constants.